src/components/common/validations/EIKValidator.tsx

```diff
@@ -0,0 +1,83 @@
+export function validateEIK9(eik: string): boolean {
+  const digits = checkInput(eik, 9)
+  const ninthDigit = calculateNinthDigitInEIK(digits)
+  return ninthDigit === digits[8]
+}
+
+export function validateEIK13(eik: string): boolean {
+  const digits = checkInput(eik, 13)
+  const thirteenDigit = calculateThirteenthDigitInEIK(digits)
+  return thirteenDigit === digits[12] && thirteenDigit !== -1
+}
+
+function calculateNinthDigitInEIK(digits: number[]): number {
+  let sum = 0
+  for (let i = 0; i < 8; i++) {
+    sum += digits[i] * FIRST_SUM_9DIGIT_WEIGHTS[i]
+  }
+  let remainder = sum % 11
```

Comment:
the compiler/linter complains about this:
18:7  Error: 'remainder' is never reassigned. Use 'const' instead.  prefer-const

please use 'yarn build' to verify

Reply:
done